docs/sources/administration/image_rendering.md

```diff
@@ -102,7 +111,84 @@ callback_url = http://localhost:3000/
 ```
 4. Restart Grafana
 
+For further information and instructions refer to [troubleshooting](#troubleshooting) and the [plugin details](https://grafana.com/grafana/plugins/grafana-image-renderer).
+
+### PhantomJS
+
+> PhantomJS is deprecated since Grafana v6.4 and will be removed in a future release. Please migrate to Grafana image renderer plugin or remote rendering service.
+
+[PhantomJS](https://phantomjs.org/) have been the only supported and default image renderer since Grafana v2.x and is shipped with Grafana.
+
+Please note that for macOS and Windows, you will need to ensure that a phantomjs binary is available under tools/phantomjs/phantomjs. For Linux, a phantomjs binary is included - however, you should ensure that any required libraries, e.g. libfontconfig1, are available.
```

Comment:
The phantomjs binary is provided for macos and windows. It's only for arm we aren't providing a phantomjs binary afaik.

Reply:
Yeah, probably really old leftover. Will fix :+1: